src/components/FeaturedStoryCard/index.js

```diff
@@ -23,21 +24,24 @@ function FeaturedStoryCard({
   return (
     <Grid container className={classes.root}>
       <Grid item xs={12} md={7}>
-        <div className={classes.media}>
-          <Image src={image} alt={title} layout="fill" />
-        </div>
+        {image && (
+          <div className={classes.media}>
+            <Image src={image} alt={title} layout="fill" />
+          </div>
+        )}
+        {!image && (
+          <RichTypography className={classes.media}>{chart}</RichTypography>
+        )}
```

Comment:
We just assume that if there is no `image`, `chart` must exists?

Reply:
@kilemensi  no. We assume chart `could` exist! `RichTypography` handles null ... Remember [this](https://github.com/CodeForAfrica/PesaYetu/pull/61#discussion_r686964916)

Reply:
Damn, someone is out for blood today!

I was talking from _design_ point of view. Why not use something like `variant`? If it's image, you pass the prop (say `vis` or something better) to `Image` and if variant is chart, you pass the same prop to `RichTypography`. This way it's clear that `FeaturedStoryCard` will show either an image or chart.